source/nodejs/adaptivecards-designer/package.json

```diff
@@ -41,25 +41,15 @@
 		"adaptive-expressions": "^4.11.0",
 		"adaptivecards": "^2.10.0",
 		"adaptivecards-templating": "^2.2.0",
-		"monaco-editor": "^0.20.0"
+		"monaco-editor": "^0.29.1"
 	},
 	"devDependencies": {
-		"@typescript-eslint/eslint-plugin": "^3.4.0",
-		"@typescript-eslint/parser": "^3.4.0",
 		"adaptive-expressions": "^4.11.0",
-		"adaptivecards": "^2.10.0",
-		"adaptivecards-templating": "^2.2.0",
+		"adaptivecards": "^3.0.0-beta.1",
```

Comment:
We don't need this change and it should stay in 2.10.x

Reply:
fixed by reverting those commits